### ISSUE TITLE
Recurrent Regression Model Fitting Problem

### ISSUE THREAD

**Issue by Pashton** [fchollet/keras#2431]
Please make sure that the boxes below are checked before you submit your issue. Thank you!

- [ y] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [y ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [y ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

Code : 

x = [numpy.random.rand(10) for v in xrange(100)]
y = [v.sum() for v in x]

model = Sequential()
model.add(LSTM(20, input_shape=(10,1)))
model.add(Dense(1))
model.add(Activation('linear'))
model.compile(optimizer='rmsprop', loss='mse')
model.fit(x,y)

The code above will give the following error:
Exception: Error when checking model input: the list of Numpy arrays that you are passing to your model is not the size the model expected. Expected to see 1 arrays but instead got the following list of 100 arrays : [ ...

Why does this happen? I have outlined my input dimension and outlined length - both match in the x.

Thank you

**Comment by Pashton** [comment]
I got this to work. Just had to change model.fit(x,y) to model.fit(numpy.asarray([x]), numpy.asarray([y])
I also changed model.add(LSTM(20, input_shape(10,1)) to model.add(LSTM(10, input_shape=(10,1))
I am getting a different error now:

![keras_error](https://cloud.githubusercontent.com/assets/1195276/14688841/8df557b0-0712-11e6-8dd8-68e75325bf68.png)

**Comment by Pashton** [comment]
Issue is resolved, it had to do with the input array. I generated x as such instead : 

x = [[numpy.random.randint(1) for v in xrange(10)] for z in xrange(100)]

That got fit to work.
